### DISCUSSION TITLE
Remove that Wall on the Remora

### DISCUSSION THREAD

**Discussion by gerakis** [FakeFishGames/Barotrauma#7687] [Ideas and suggestions]
This wall has got to go. Every single multiplayer map I've been on everyone just plasma cutters the wall down and yells at anyone who repairs it. Just get rid of it please. 

I get that subs have to have some problems or the round would be boring, but this is more of an ergonomics thing. If anything it just makes the sub more prone to flooding so it creates even more chaos per round. It greatly improves the quality of life for anyone using the sub if the wall is missing.

This seems like it would be a total nightmare if you had bots. They would try to repair the wall even though you don't want them to.

![remove that wall](https://user-images.githubusercontent.com/16128947/145438915-3ab17c5c-7188-4553-8732-d7f6e2e57ad1.png)

**Comment by MaaiG** [comment]
Vanilla boats have a lot of flaws. It gives them an opportunity to improve. If they’re "perfect" you won’t have anything to do and you won’t build your submarine. Building your own boat - that is half the game. Maybe much more! ))

Perhaps you suggest adding a tool for permitted wall demolition? This is a different case )) - "doorway" tool! 👍

**Comment by Terragen** [comment]
I don't think bots will try to repair the wall if you issue the "ignore" order on it.

**Reply by ghost** [reply]
You cannot order bots to ignore walls, only interactable items.

**Reply by Terragen** [reply]
![image](https://user-images.githubusercontent.com/4472968/146550742-cbbf725a-708d-41ac-a7c5-512459c31a52.png)

It is possible to issue the "ignore" command on walls.
